feat: adding extension files back in and readmes for each template (#40)
apollographql/rover-init-starters: README.md

👋 Welcome!

> This repository is used to initialize by `rover init` to curate an API orchestration project using [Apollo’s Rover CLI](https://www.apollographql.com/docs/rover) by providing a structured setup for managing and composing APIs efficiently.

Reviewer: I would suggest reducing the complexity of this first sentence.

- Use "graph" vs. "API orchestration project", then include "orchestrating" at the end
- Split opening sentence into two for increased readability.
- Use active rather than passive voice

```suggestion
[Apollo’s Rover CLI](https://www.apollographql.com/docs/rover) uses this repository to initialize a [graph](https://www.apollographql.com/docs/concepts/graphs). Running Rover's [`init`](https://www.apollographql.com/docs/rover/commands/init) command provides a structured setup for managing and orchestrating APIs efficiently.
```
